ultralytics/data/augment.py

```diff
@@ -541,6 +541,7 @@ def __init__(self, dataset, imgsz=640, p=1.0, n=4):
         self.imgsz = imgsz
         self.border = (-imgsz // 2, -imgsz // 2)  # width, height
         self.n = n
+        self.buffer_enabled = self.dataset.cache != "ram"
 
     def get_indexes(self, buffer=True):
```

Comment:
@Y-T-G we already have a boolean argument here to enable or disable. We don't want duplicate variables here, we want to use the existing variable that serves this purpose.

Reply:
Ok I looked around and it looks like the argument is not used anywhere, maybe we should remove the method arg then. I'll apply this change.